crates/core/tedge/src/cli/reconnect/cli.rs

```diff
@@ -0,0 +1,45 @@
+use crate::cli::common::Cloud;
+use crate::cli::connect::CommonMosquittoConfig;
+use crate::command::*;
+use tedge_config::system_services::service_manager;
+
+use super::command::ReconnectBridgeCommand;
+
+const C8Y_CONFIG_FILENAME: &str = "c8y-bridge.conf";
+const AZURE_CONFIG_FILENAME: &str = "az-bridge.conf";
+
+#[derive(clap::Subcommand, Debug)]
+pub enum TEdgeReconnectBridgeCli {
+    /// Remove bridge connection to Cumulocity.
+    C8y,
+    /// Remove bridge connection to Azure.
+    Az,
+}
+
+impl BuildCommand for TEdgeReconnectBridgeCli {
+    fn build_command(self, context: BuildContext) -> Result<Box<dyn Command>, crate::ConfigError> {
+        let cmd = match self {
+            TEdgeReconnectBridgeCli::C8y => ReconnectBridgeCommand {
```

Comment:
Since majority of the fields are the same for both clouds, I'd move this `ReconnectBridgeCommand` instantiation outside the match block and only initialise the cloud specific parameters like `config_file`, `cloud` and `use_agent` inside the match block. That'll better highlight the aspects that are different between the clouds.

Reply:
Done